imports/plugins/core/versions/server/migrations/4_update_templates_priority.js

```diff
@@ -0,0 +1,40 @@
+import { Migrations } from "/imports/plugins/core/versions";
+import { Packages } from "/lib/collections";
+
+const query = { layout: { $type: 3 } }; // gets docs with layout objects
```

Comment:
Ok..cool.

So... instead of making a list of packages, which might not capture all that is required based on the time the migration is run, how about this:

```const query = {
  name: { $regex: /reaction-/ }, // picks reaction pkgs
  layout: { $type: 3 } // docs with layouts set
};```